### ISSUE TITLE
[-Wignored-qualifiers] warning in tidy

### ISSUE THREAD

**Issue by Girgias** [htacg/tidy-html5#866]
I am not exactly sure if it is bug in tidy5 itself or in how Ubuntu packages it. But adding some compiler warning into php/php-src#5151 two [-Wignored-qualifiers] came out from the tidy header:

```
In file included from /home/vsts/work/1/s/ext/tidy/tidy.c:30:0:
/usr/include/tidy/tidy.h:2095:13: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
 TIDY_EXPORT const ctmbstr TIDY_CALL TidyLangWindowsName( const tidyLocaleMapItem *item );
             ^~~~~
/usr/include/tidy/tidy.h:2101:13: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
 TIDY_EXPORT const ctmbstr TIDY_CALL TidyLangPosixName( const tidyLocaleMapItem *item );
             ^~~~~
```

The relevant C code within the Tidy extension is:
```c
#if HAVE_TIDY_H
#include "tidy.h"
#elif HAVE_TIDYP_H
#include "tidyp.h"
#endif
```

And the Ubuntu package used is [``libtidy-dev``](https://packages.ubuntu.com/bionic/libtidy-dev).

This seems to be the same issue as reported in #746 which was closed last year, therefore it may also be that the Ubuntu package is out of date.

Best regards.

**Comment by Girgias** [comment]
Small bump

**Comment by geoffmcl** [comment]
@Girgias, sorry for the long delay...

But as you point out this is the **SAME** as issue #746, which was **CLOSE** on the merge of PR #747, circa Jan 2019, and is thus **FIXED** since tidy version 5.7.20...

We have little control over how quickly distributions pick up on this...

Accordingly, closing this now...

**Comment by Girgias** [comment]
No worries, :) we got around it by disabling the compiler flag for the extension.
